dash/backend/src/modules/falco/controllers/falco.controller.ts

```diff
@@ -59,15 +59,13 @@ export class FalcoController {
         @Query('pod') pod?: string,
         @Query('image') image?: string,
         @Query('signature') signature?: string,
-        @Query('eventId') eventId?: number
-    ): Promise<{ logCount: number, list: FalcoDto[]}>
+    ): Promise<{ logCount: number, list: FalcoDto[],  csvLogList: FalcoDto[]}>
     {
         // default values - seems to be filling with NaN instead of null which causes the defaults in Dao to not work right
         if (!limit) limit = null;
         if (!page) page = null;
-        if (!eventId) eventId = null;
 
-        return this.falcoService.getFalcoLogs(clusterId, limit, page, priorities, orderBy, startDate, endDate, namespace, pod, image, signature, eventId);
```

Comment:
Why was this parameter removed? Is it ever used anywhere?

Reply:
As I was making changes to getFalcoLogs, I realized that the eventId was never used in the query, not event passed from the frontend. Event id is essentially the primary id. We already have the desc/ asc filter to cover the sorting. I think it's safe to remove something we don't use.